src/routes/dev/data-graphic/basic.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import {
```

Comment:
Not sure what's happening here but I just see this when I mount this component on an empty page

<img width="808" alt="Screenshot 2022-07-06 at 8 08 16 PM" src="https://user-images.githubusercontent.com/4402679/177576618-8ed9a306-9191-4ae4-9a62-e0214b4099e2.png">